src/ParticleDA.jl

```diff
@@ -366,12 +366,16 @@ function init_filter(filter_params::FilterParameters, model_data, nprt_per_rank:
 
     model_noise_params = get_model_noise_params(model_data)
     obs_noise_std = get_obs_noise_std(model_data)
+    # Precompute two FFT plans, one in-place and the other out-of-place
+    C = complex(T)
+    tmp_array = Matrix{C}(undef, grid_ext.nx, grid_ext.ny)
+    fft_plan, fft_plan! = FFTW.plan_fft(tmp_array), FFTW.plan_fft!(tmp_array)
 
-    offline_matrices = init_offline_matrices(grid, grid_ext, stations, model_noise_params, obs_noise_std, T)
+    offline_matrices = init_offline_matrices(grid, grid_ext, stations, model_noise_params, obs_noise_std, fft_plan, fft_plan!, T)
```

Comment:
It looks like `fft_plan, fft_plan!` are always passed together so could just store them as a tuple?

Reply:
This is actually what I did at the beginning, but I didn't like it too much because it was opaque what `fft_plan[1]` and `fft_plan[2]` were (which is the out-of-place, which is the in-place?).  I guess you'd suggest a named tuple? :smile:

Reply:
😁  Not a big deal either way